dev/io.openliberty.netty.internal.impl/src/io/openliberty/netty/internal/impl/NettyFrameworkImpl.java

```diff
@@ -304,37 +317,53 @@ public ChannelFuture start(BootstrapExtended bootstrap, String inetHost, int ine
         return UDPUtils.start(this, bootstrap, inetHost, inetPort, bindListener);
     }
 
+    @Override
+    public ChannelFuture startOutbound(BootstrapExtended bootstrap, String inetHost, int inetPort,
+            ChannelFutureListener bindListener) throws NettyException {
+        if (bootstrap.getConfiguration() instanceof TCPConfigurationImpl) {
+            return TCPUtils.startOutbound(this, bootstrap, inetHost, inetPort, bindListener);
+        } else {
+            return UDPUtils.startOutbound(this, bootstrap, inetHost, inetPort, bindListener);
+        }
+    }
+
     @Override
     public ChannelFuture stop(Channel channel) {
-        if (isActive && channel.isOpen()) {
-            ChannelFuture closeFuture = channel.close();
-            activeChannels.remove(closeFuture.channel());
-            closeFuture.addListener(future -> {
-                if (!future.isSuccess()) {
-                    if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
-                        String channelName = channel.attr(ConfigConstants.NameKey).get();
-                        Tr.debug(tc,
-                                "channel " + channelName + " failed to stop due to: " + future.cause().getMessage());
+        if (activeChannels.contains(channel)) {
+            activeChannels.remove(channel);
+            if (isActive && channel.isOpen()) {
```

Comment:
the old code appeared careful to get rid of any prior listener (in L:311) do we not need that here?